source/_components/alarm_control_panel.paradox.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Paradox Alarm Control Panel"
+description: "Instructions how to integrate Paradox alarms into Home Assistant."
+date: 2016-10-07 12:17
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: paradox.png
+ha_category: Alarm
+ha_release: 0.30
```

Comment:
Won't be merged into the coming release this close to the release so number need to change to `0.31`.